code/modules/surgery/bodyparts/head_hair_and_lips.dm

```diff
@@ -44,7 +32,7 @@
 		else
 			show_debrained = FALSE
 
-		if(!owner.get_organ_slot(ORGAN_SLOT_EYES))
+		if(!eyes_hidden && !owner.get_organ_slot(ORGAN_SLOT_EYES))
```

Comment:
eyeless is for missing eyes, not "dont render eyes", iirc

Reply:
yes this is to make eyeless overlay not render

Reply:
because if eyes shouldnt render the eyeless overlay shouldnt either probably

Reply:
don't you want OR and not AND

Reply:
no i want the eyeless overlay to render if your eyes arent hidden and oyu dont have eyes

Reply:
kk